go/cron/window_batch_task_test.go

```diff
@@ -332,3 +332,45 @@ func TestAcctCleanup(t *testing.T) {
 		},
 	)
 }
+
+func TestNodeCurrentSessionCleanup(t *testing.T) {
+	testWindowSqlCleanup(
+		t,
+		"node_current_session_cleanup",
+		map[string]interface{}{
+			"timeout": 10.0,
+			"batch":   100.0,
+			"window":  float64(12 * 60 * 60),
+		},
+		[]string{
+			"DELETE FROM node_current_session",
+			`INSERT INTO node_current_session (mac, updated, last_session_id, is_online) VALUES
```

Comment:
Adjust the test, or add another test to assert test `updated` and `last_session_id` are overwritten on duplicate key